src/Microsoft.Health.Fhir.Anonymizer.R4.CommandLineTool/configuration-sample.json

```diff
@@ -197,7 +197,7 @@
     {"path": "nodesByType('Address')", "method": "redact"},
     {"path": "nodesByType('Age')", "method": "redact"},
     {"path": "nodesByType('Annotation')", "method": "redact"},
-    {"path": "nodesByType('HumanName')", "method": "redact"},
+    {"path": "nodesByType('HumanName')", "method": "presidio"},
```

Comment:
Can we leave the human name as redact as it was?
We should have a node type for free text. You can use `text` as seen here: https://www.hl7.org/fhir/patient-example.json.html

Reply:
pushed the json example to the repo, added a presidio method on the 'text' json field.